### DISCUSSION TITLE
How to calculate HEI-2015

### DISCUSSION THREAD

**Discussion by binyam-1** [jamesjiadazhan/dietaryindex#129] [Q&A] [answered]
Dear James,

Thank you for developing the package and for the time and effort you have invested in it. I am writing to inquire if the package is able to calculate the Healthy Eating Index (HEI) based on two days of dietary recall data from NHANES. I would appreciate any information you can provide on this matter.

Thank you in advance for your assistance.

Best regards,
Binyam Girma

**Comment by jamesjiadazhan** [comment, marked as answer]
Thanks for reaching out, Binyam. The package just takes the data and gives you the result. So, yes, I think it is possible for the package to calculate HEI-2015 based on 2 days of dietary recall, but keep in mind, the official SAS code from USDA only uses the first day of dietary recall data and by using 2 days of dietary recall, you can reduce final results' sample size moderately, since some NHANES participants didn't complete both visits.

With that being said, you can use the HEI2015_NHANES_FPED functions twice and then merge them to accomplish what you proposed. For each use, you just input NUTRIENT and FPED for each specific visit while using the same DEMO. After having 2 HEI2015 results, you just use inner_join to merge them based on SEQN, and then take the average of 2 HEI2015 results. I think it would work. Let me know if it's not.
